docs/source/markdown/podman-events.1.md

```diff
@@ -10,7 +10,7 @@ podman\-events - Monitor Podman events
 
 Monitor and print events that occur in Podman. Each event will include a timestamp,
 a type, a status, name (if applicable), and image (if applicable).  The default logging
-mechanism is *journald*. This can be changed in libpod.conf by changing the `events_logger`
+mechanism is *journald*. This can be changed in containers.conf by changing the `events_logger`
```

Comment:
Probably should add container.conf(5) to the bottom of this page.

Reply:
Done